00 Base/src/components/myComponent/myComponent.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+// import { render, cleanup } from '@testing-library/react';
+import {
+  render,
+  fireEvent,
+  waitForElement,
+  act,
+  RenderResult,
+} from '@testing-library/react';
+//import * as myApi from '../myApi';
```

Comment:
Please delete all comented code

Reply:
Try to increase the coverad¡ge here https://github.com/dailymp/samples-for-react-testing-library-training/pull/11#issue-402789913